ojd_daps_skills/pipeline/extract_skills/extract_skills.py

```diff
@@ -31,9 +31,18 @@ class ExtractSkills(object):
     :param verbose: Whether to limit the number of logging messages (True) or not (False, good for debugging), defaults to True
     :type verbose: bool
 
+    :param multi_process: Whether to use multiprocessing (True) or not (False), defaults to True
+    :type multi_process: bool
+
     """
 
-    def __init__(self, config_name="extract_skills_toy", local=True, verbose=True):
+    def __init__(
+        self,
+        config_name="extract_skills_toy",
+        local=True,
+        verbose=True,
+        multi_process=True,
```

Comment:
should we default to true givens @Jack-Vines 's troubles with it?

Reply:
good point!

Reply:
done